syncdb.drush.inc

```diff
@@ -133,8 +133,8 @@ function drush_syncdb($source) {
  *   List of table names within the path.
  */
 function _syncdb_parallel_import($local_dump_path) {
-  $db = drush_get_option('database');
-  drush_op_system("find " . $local_dump_path . " -name '*sql' | parallel --use-cpus-instead-of-cores --jobs 200% -v drush sql-query --database=$db --file={}");
+  $cmd = drush_find_drush(). ' '. _drush_backend_generate_command([], 'sql-query', [], drush_redispatch_get_options() + ['strict' => 0]);
```

Comment:
Minor / cosmetic: white space around concatenation doesn't seem to follow Drupal coding standards. :)